Detectors/TRD/base/include/TRDBase/Digit.h

```diff
@@ -50,61 +45,19 @@ class Digit
   void setDetector(int det) { mDetector = det; }
   void setRow(int row) { mRow = row; }
   void setPad(int pad) { mPad = pad; }
-  void setADC(ArrayADC_t adc) { mADC = adc; }
+  void setADC(ArrayADC adc) { mADC = adc; }
   // Get methods
   int getDetector() const { return mDetector; }
   int getRow() const { return mRow; }
   int getPad() const { return mPad; }
-  ArrayADC_t getADC() const { return mADC; }
+  ArrayADC getADC() const& { return mADC; }
```

Comment:
It should be `ArrayACD const& getADC() cont {return mADC;}`. Same everywhere else. For instance a few lines up `setADC(ArrayADC const& adc) { mADC = adc; }`

Reply:
Thanks, fixed now.